docs/index.html

```diff
@@ -101,6 +101,62 @@
             '/': 'Search',
           },
         },
+        vueComponents: {
+          'button-counter': {
+            template: `
+            <button @click="count += 1">
+              You clicked me {{ count }} times
+            </button>
+          `,
+            data() {
+              return {
+                count: 0,
+              };
+            },
+          },
+        },
+        vueGlobalOptions: {
```

Comment:
I'm guessing these `vueGlobalOptions` get merged with each component in `vueComponents`.

Reply:
Not shared. Here's the description:

> ## vueGlobalOptions
> 
> - type: `Object`
> 
> Specifies Vue options to be shared throughout your site. These options will be used when Docsify detects Vue content in the main content area that has not been previously mounted via `vueMountOptions`, `vueComponents`, or a markdown `<script>`.

Reply:
@trusktr Updated the description as follows:

> Specifies options for use with Vue content not explicitly mounted with `vueMounts`, `vueComponents`, or a markdown `<script>`. Changes to global `data` will persist and be reflected anywhere global references are used.

Hopefully that makes it more clear.

Reply:
That is indeed a lot more clear than before.

The object looks like a Vue component definition. What if it is called `vueGlobalComponent` instead?

Also is there an example?

Reply:
Nvm, the `vue.md` file was not expanded by default here in the diff view, so searching for `vueGlobalOptions` didn't yield results but now it does.

How is it that we can change the DOM on page changes, and the data still works with each new DOM?

Reply:
Nvm, the source answers my question.